build/_build.csproj.DotSettings

```diff
@@ -0,0 +1,27 @@
+<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Do we need that file at all? Looks like some Rider specific file to configure project local values.

Reply:
According to [Nuke's documentation](https://nuke.build/docs/getting-started/setup/), it indeed is a ReSharper/Rider formatting file, so removing it shouldn't break the workflow. 

However, I don't know whether if it should be removed or not. On [dd-trace-dotnet](https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/build/_build/_build.csproj.DotSettings), they aren't removing it.

Reply:
Removed :+1: